modules/nw-egress-ips-config-object.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
+
+[id="nw-egress-ips-config-object_{context}"]
+= EgressIPconfig object
```

Comment:
Small nit, but the capitalization isn't consistent between the title and the body text. Should this be `egressIPConfig object`?